modules/@apostrophecms/notification/ui/apos/apps/ApostropheNotification.js

```diff
@@ -94,10 +96,11 @@ export default function() {
           });
 
           this.notifications = [...this.notifications, ...(notifications || [])];
+          this.dismissed = [...this.dismissed, ...(dismissed || [])];
```

Comment:
So you are just going to keep all the dismissed notifications you've seen in RAM. I think that's reasonable for the lifetime of the page! Later we could optimize this to just retain those that have the latest modification time.